src/Symfony/Component/Form/Tests/EnumFormTypeGuesserTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\EnumFormTypeGuesser;
+use Symfony\Component\Form\Extension\Core\Type\EnumType as FormEnumType;
+use Symfony\Component\Form\Guess\Guess;
+use Symfony\Component\Form\Guess\TypeGuess;
+use Symfony\Component\Form\Guess\ValueGuess;
+use Symfony\Component\Form\Tests\Fixtures\BackedEnumFormTypeGuesserCaseEnum;
+use Symfony\Component\Form\Tests\Fixtures\EnumFormTypeGuesserCase;
+use Symfony\Component\Form\Tests\Fixtures\EnumFormTypeGuesserCaseEnum;
+
+final class EnumFormTypeGuesserTest extends TestCase
```

Comment:
final is pure visual overhead ;)

```suggestion
class EnumFormTypeGuesserTest extends TestCase
```